scalardl-test/src/main/java/scalardl/Common.java

```diff
@@ -26,17 +30,29 @@
   public static ClientConfig getClientConfig(Config config) {
     String host = config.getUserString("client_config", "dl_server", HOST);
     String port = config.getUserString("client_config", "dl_server_port", PORT);
+    String auditorEnabled = config.getUserString("client_config", "auditor_enabled", AUDITOR_ENABLED);
+    String auditorHost = config.getUserString("client_config", "auditor_host", AUDITOR_HOST);
+    String auditorPort = config.getUserString("client_config", "auditor_port", AUDITOR_PORT);
     String certificate = config.getUserString("client_config", "certificate");
     String privateKey = config.getUserString("client_config", "private_key");
 
     Properties properties = new Properties();
     properties.setProperty(ClientConfig.SERVER_HOST, host);
     properties.setProperty(ClientConfig.SERVER_PORT, port);
+    properties.setProperty(ClientConfig.AUDITOR_ENABLED, auditorEnabled);
+    properties.setProperty(ClientConfig.AUDITOR_HOST, auditorHost);
+    properties.setProperty(ClientConfig.AUDITOR_PORT, auditorPort);
     properties.setProperty(ClientConfig.CERT_HOLDER_ID, CERT_HOLDER_ID);
     properties.setProperty(ClientConfig.CERT_PATH, certificate);
     properties.setProperty(ClientConfig.PRIVATE_KEY_PATH, privateKey);
 
-    return new ClientConfig(properties);
+    ClientConfig clientConfig = null;
+    try {
+      clientConfig = new ClientConfig(properties);
+    } catch (IOException e) {
+      throw new IllegalArgumentException(e.getMessage());
+    }
+    return clientConfig;
   }
 
   public static ClientService getClientService(Config config) {
```

Comment:
We can use `ClientServiceFactory` instead of using Guice directly.
https://github.com/scalar-labs/scalar/blob/master/client/src/main/java/com/scalar/dl/client/service/ClientServiceFactory.java